### ISSUE TITLE
[Question] pretain 7B loss doesn't change

### ISSUE THREAD

**Issue by aprilehannibal** [haotian-liu/LLaVA#169]
### Question

I tried to pretrain llava-7b on 8 * A100(40G), CUDA117, torch==1.13.1, py3.10
 I found the loss is doesn't change when training.

<img width="547" alt="image" src="https://github.com/haotian-liu/LLaVA/assets/14832463/8ea0d73f-103a-4aae-a146-cbba3d040b0d">

here is my pretraining scripts:
`torchrun --nnodes=1 --nproc_per_node=8 --master_port=25001 \
    llava/train/train_mem.py \
    --model_name_or_path ./checkpoints/llama_vicuna-7b-delta-v1.1 \
    --data_path /path/to/LLaVA-CC3M-Pretrain-595K/chat.json \
    --image_folder /path/to/cc3m_595k_images \
    --vision_tower ./openai/clip-vit-large-patch14 \
    --tune_mm_mlp_adapter True \
    --mm_vision_select_layer -2 \
    --mm_use_im_start_end \
    --bf16 True \
    --output_dir ./checkpoints/llava-7b-pretrain \
    --num_train_epochs 1 \
    --per_device_train_batch_size 2 \
    --per_device_eval_batch_size 1 \
    --gradient_accumulation_steps 8 \
    --evaluation_strategy "no" \
    --save_strategy "steps" \
    --save_steps 2400 \
    --save_total_limit 1 \
    --learning_rate 2e-3 \
    --weight_decay 0. \
    --warmup_ratio 0.03 \
    --lr_scheduler_type "cosine" \
    --logging_steps 1 \
    --tf32 True \
    --model_max_length 2048 \
    --gradient_checkpointing True \
    --lazy_preprocess True`

**Comment by aprilehannibal** [comment]
I wonder the LLM you used for pretraining is the  "lmsys/vicuna-7b-delta-v1.1" or the origin "llama 7b" weight.

**Comment by haotian-liu** [comment]
Hi @aprilehannibal, you need to perform [weight conversion](https://github.com/lm-sys/FastChat#vicuna-7b) for vicuna weights, by combing llama-7b and vicuna-delta -> vicuna-7b.  We pretrain based on vicuna-7b.

We'll update the instruction to make it clearer, thanks.

**Comment by aprilehannibal** [comment]
OK, got it! Thanks a lot!
